Error: Cannot find module 'emoji-regex'

If I want to run prettier in my library, I get the following error:

```
$ yarn format
yarn run v1.12.3
$ prettier --write src/**/*.ts
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'emoji-regex'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/schmidsi/Development/@melonproject/protocol/node_modules/prettier-plugin-solidity/src/prettier-comments/common/util.js:4:20)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
error Command failed with exit code 1.
```

It suppose that the error comes from this line of code:

https://github.com/prettier-solidity/prettier-plugin-solidity/blob/7e89009fbd88b9c7e861be125f056ac3eee4c9fa/src/prettier-comments/common/util.js#L4

A possible fix was to install 'emoji-regex' in the consuming project, but that is a bad work around. You could probably remove this dependency completely?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "prettier-plugin-solidity",
-  "version": "1.0.0-alpha.11",
+  "version": "1.0.0-alpha.12",
   "description": "prettier plugin for solidity",
   "main": "src",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -46,17 +46,20 @@
   "homepage": "https://github.com/prettier-solidity/prettier-plugin-solidity#readme",
   "devDependencies": {
     "codecov": "^3.0.4",
-    "eslint": "^5.3.0",
+    "eslint": "^5.9.0",
     "eslint-config-airbnb": "^17.0.0",
     "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
     "eslint-plugin-react": "^7.11.0",
     "jest": "^23.5.0",
     "jest-watch-typeahead": "^0.2.0"
   },
   "dependencies": {
-    "extract-comments": "^1.0.0",
-    "prettier": "^1.14.2",
-    "solidity-parser-antlr": "^0.3.1"
+    "emoji-regex": "^7.0.1",
+    "escape-string-regexp": "^1.0.5",
+    "extract-comments": "^1.1.0",
+    "prettier": "^1.15.2",
+    "solidity-parser-antlr": "^0.3.1",
+    "string-width": "^2.1.1"
   }
 }

